css/templates/land-see/page.css

```diff
@@ -566,3 +567,156 @@ h4,
   width: 44px;
   height: 35px;
 }
+
+/* Hero Carousel */
+
+.land-see-hero-carousel {
+  max-width: 800px;
+}
+
+.land-see-hero-title,
+.land-see-hero-caption,
+.land-see-hero-button {
+  color: var(--color-font-secondary);
+}
+
+.land-see-hero-content amp-img:before {
+  content: '';
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: linear-gradient(to bottom, rgba(125,126,125,0) 0%,rgba(125,126,125,0.15) 30%,rgba(14,14,14,0.7) 100%);
+  z-index: 1;
+}
+
+.land-see-hero-image {
+  background-color: rgba(125,126,125,1);
+}
+
+.land-see-cover-position-top img {
+  object-position: top center;
+}
+
+.land-see-cover-position-bottom img {
+  object-position: bottom center;
+}
+
+.land-see-hero-typography {
+  bottom: -7%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.land-see-hero-typography a {
+  text-decoration: none;
+}
+
+.land-see-hero-title {
+  font: 400 var(--h1)/var(--line-height-6) var(--font-primary);
+  letter-spacing: normal;
+}
+
+.land-see-hero-caption {
+  width: 28rem;
+  font: 400 var(--h7)/var(--line-height-6) var(--font-primary);
```

Comment:
i think for a lot of these you would be fine overriding variables or rewriting the typography css classes (not sure if you had already explored that)